modules/lib/maintainers.nix

```diff
@@ -13,6 +13,16 @@
     github = "amesgen";
     githubId = 15369874;
   };
+  anthonyroussel = {
+    email = "anthony@roussel.dev";
+    github = "anthonyroussel";
+    githubId = 220084;
+    name = "Anthony Roussel";
+    keys = [{
+      longKeyId = "ed25519/0x9DC4987B1A55E75E";
+      fingerprint = "472D 368A F107 F443 F3A5  C712 9DC4 987B 1A55 E75E";
+    }];
+  };
```

Comment:
You already seem to be listed in the Nixpkgs maintainers list so then there is no need to be here as well.
```suggestion
```